src/app/store/store.js

```diff
@@ -1,18 +1,50 @@
 import { writable, derived, get } from 'svelte/store';
 import produce from 'immer';
 import telemetrySearch from './telemetry-search';
+import { getProbe } from './api';
+
+import CONFIG from '../config.json';
+
+import { weightedQuantile } from '../../utils/stats';
+
+export function getFieldValues(fieldKey) {
+  return CONFIG.fields[fieldKey].values;
+}
+
+export function isField(fieldKey) {
+  return Object.keys(CONFIG.fields).includes(fieldKey);
+}
+
+export function getFieldValueMetadata(fieldKey, valueKey) {
+  return getFieldValues(fieldKey).find((v) => v.key === valueKey);
+}
+
+export function isValidFieldValue(fieldKey, valueKey) {
+  return getFieldValues(fieldKey).map((fv) => fv.key).includes(valueKey);
+}
+
+export function getFieldValueLabel(fieldKey, valueKey) {
+  return getFieldValueMetadata(fieldKey, valueKey).label;
+}
+
+export function getDefaultFieldValue(fieldKey) {
+  return getFieldValues(fieldKey)[0].key;
+}
 
 const initStore = {
   probe: {
     name: undefined,
+    apiName: undefined,
     description: undefined,
     audienceSize: 0,
     totalSize: 0,
   },
-  product: 'all',
-  channel: 'all',
-  os: 'all',
+  product: 'Firefox',
+  channel: getDefaultFieldValue('channel'),
+  os: getDefaultFieldValue('os'),
+  version: 69,
```

Comment:
Would this be a TODO to fetch the latest firefox versions for each channel? Is that info we should keep in firestore?

Reply:
yep, will add a `TODO` here. I haven't figured out how we should handle the versioning quite yet.